src/OpenDiffix.Core/Planner.fs

```diff
@@ -46,7 +46,7 @@ let rec private collectSetFunctions expression =
 
 let private planJoin join columnIndices =
   let leftColumnCount = QueryRange.columnsCount join.Left
-  let leftIndices, rightIndices = List.partition ((>=) leftColumnCount) columnIndices
+  let leftIndices, rightIndices = List.partition ((>) leftColumnCount) columnIndices
```

Comment:
bugfix

Reply:
`(>) leftColumnCount` is very confusing because it actually means `fun x -> x < leftColumnCount`. Can we please expand to full form?

Reply:
You're right! I also got confused by that, because the docs for `partition` say it returns: `A list containing the elements for which the predicate evaluated to false and a list containing the elements for which the predicate evaluated to true.`.

But it actually is the other way around! 

The description has it right `two collections, containing the elements for which the given predicate returns True and False respectively` :exploding_head:

Reply:
it's fixed in the fresh {version of docs](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-collections-listmodule.html)

Reply:
`[1; 2; 3] |> List.map((-) 1)` results in `[0; -1; -2]` because the args are the other way around!